.ci/kitchen-macosxhighsierra-py3

```diff
@@ -10,10 +10,10 @@ runTestSuite(
     jenkins_slave_label: 'kitchen-slave-mac',
     kitchen_platforms_file: '/var/jenkins/workspace/pre-golden-nox-platforms.yml',
     kitchen_verifier_file: '/var/jenkins/workspace/nox-verifier.yml',
-    nox_env_name: 'runtests-zeromq',
+    nox_env_name: 'pytest-zeromq',
     nox_passthrough_opts: '',
     python_version: 'py3',
-    run_full: params.runFull,
+    //splits: ['unit', 'integration', 'multimaster'],
```

Comment:
Why is this just commented out instead of either live or completely removed? There are several of these lines.

Reply:
Ran into issues when running split at some point in time and just commented it out.
I can remove on the final commit.

Reply:
I'm actually leaving it in since they are not all equal and getting them to this point required some trial and error which I don't want to go through again, for when we split it up again.